src/software/ai/hl/stp/play/enemy_ball_placement_play.cpp

```diff
@@ -81,8 +81,9 @@ void EnemyBallPlacementPlay::ballPlacementWithShadow(
             move_tactics[0]->updateControlParams(
                 placement_point + placement_to_net,
                 placement_to_net.orientation() + Angle::half(), 0);
-            shadow_enemy->updateControlParams(enemy_threats.at(0),
-                                              ROBOT_MAX_RADIUS_METERS * 3);
+            // We need a big shadow distance to avoid "bullying" the robot ball placing.
+            // Otherwise, we get a penalty
+            shadow_enemy->updateControlParams(enemy_threats.at(0), 1.25);
```

Comment:
wheres the 1.25 from? maybe pull it out to a constant

Reply:
oops that was supposed to get changed to something else - will change and comment